container/falcon-container-terraform/demo

```diff
@@ -0,0 +1,87 @@
+# Looks for variables file to accomodate unique suffix
+[ -f ~/TF_variables ] && source ~/TF_variables
+
+# Load the cloud shell public IP address into a variable to use in NSG
+export TF_VAR_your_ip_address="$(curl -s ifconfig.co)/32"
+
+# Create ssh key for admin vm access
+export TF_VAR_ssh_key=~/.ssh/container_lab_ssh_key
+if ! [ -f "$TF_VAR_ssh_key" ]; then
+    ssh-keygen -t rsa -b 2048 -N '' -f "$TF_VAR_ssh_key"
+fi
+export TF_VAR_ssh_key_pub=${TF_VAR_ssh_key}.pub
+
+# Get the logged in users objectId to use for keyvault assignment
+export TF_VAR_object_id=$(az ad signed-in-user show | jq '.objectId' | sed 's/\"//g')
+
+# Work through collecting needed variables if they are not avaiable
+[ -z "$TF_VAR_suffix" ] && read -p 'Enter a unique suffix value: ' TF_VAR_suffix && \
+      export TF_VAR_suffix
+[ -z "$TF_VAR_falcon_client_id" ] && read -p 'Enter the Falcon API Client ID: ' TF_VAR_falcon_client_id && \
+      export TF_VAR_falcon_client_id
+[ -z "$TF_VAR_falcon_client_secret" ]  && read -p 'Enter the Falcon API Client Secret: ' TF_VAR_falcon_client_secret && \
+      export TF_VAR_falcon_client_secret
+[ -z "$TF_VAR_falcon_cid" ] && read -p 'Enter the Falcon CID ' TF_VAR_falcon_cid && \
+      export TF_VAR_falcon_cid
+[ -z "$TF_VAR_falcon_cloud" ] && read -p 'Enter the Falcon Platform cloud region: ' TF_VAR_falcon_cloud && \
+      export TF_VAR_falcon_cloud
+
+# Save the unique suffix for later
+cat << EOF > ~/TF_variables
+export TF_VAR_suffix=$TF_VAR_suffix
+EOF
+
+# Main demo up function
+demo_up() {
+    BRANCH=master
+    GITHUB_ORG=CrowdStrike
+    
+    [ -d ~/cloud-azure ] || (cd "$HOME" && git clone --branch $BRANCH --depth 1 https://github.com/$GITHUB_ORG/cloud-azure)
+    [ -d ~/falcon-container-terraform ] || (ln -sf $HOME/cloud-azure/container/falcon-container-terraform $HOME/falcon-container-terraform)
+    cd ~/falcon-container-terraform
+
+    terraform init 
+    terraform apply
+
+cat <<__END__
+                _ _
+                (_) |             Your kubernetes cluster,
+    __      ____ _ _| |_            Your admin vm,
+    \ \ /\ / / _\` | | __|           Your Falcon Container Sensor,
+     \ V  V / (_| | | |_            and Your vulnerable application,
+      \_/\_/ \__,_|_|\__|           are all coming up.
+
+                                    This can take about 5 minutes.
+    
+                                    Do not allow the console to time out!
+__END__
+
+    sleep 330
```

Comment:
Just wondering whether it really takes 5.5 minutes for azure to spin-up a vm.

We could do a bit of active wait like:
```
while ! terraform output vm_public_ip | grep SOME_IP_REGEXP; do 
    sleep 10
done

while ! ssh  -o "StrictHostKeyChecking no" azadmin@$(terraform output vm_public_ip | tr -d '"') -i ~/.ssh/container_lab_ssh_key true > /dev/null 2>&1; do
    sleep 10
done
```

Reply:
This is something we can bring down now that I worked in the LIVE_LOG.

Reply:
I will work on an active loop

Reply:
The azure-cli takes forever to install and for a while it was taking up to 15 minutes after this latest release. That is really a variable on time here. Before the LIVE_LOG integration it was not a very good working solution.

Reply:
Changed to 60 seconds